app/workers/delivery_request_worker.rb

```diff
@@ -16,12 +14,12 @@ def perform(email_id, metrics, queue)
       return
     end
 
-    attempt = DeliveryRequestService.call(
+    increment_rate_limiter
```

Comment:
Yup was intentional, not mentioning it was not - I'll amend the commit.

Reply:
I had a quick look, but can't see anything about it. Perhaps it got lost in another amend?

Reply:
I've added details about this in the commit message. It was actually a bigger can of worms than I originally realised because of me missing that darn puppet env var. As we didn't increment the rate limit unless we made an attempt staging/integration would have very low numbers here (as must of their attempts are aborted due to https://github.com/alphagov/email-alert-api/blob/166834e3a1b78a3f427f7684ce98121f89a80bf6/app/services/delivery_request_service.rb#L19. Therefore with this being pre-emptive rather than retrospective we need staging/integration to have similar rate limit otherwise they'll be rate limiting on each publishing. 

My takeaway from this was that the rate limit was rather oddly coupled, as if it applies to notify only it should be part of the NotifyProvider whereas if it is at worker level it should be for all emails regardless of whether they're sent or provider.